src/test/scala/de/zalando/beard/renderer/BeardTemplateRendererSpec.scala

```diff
@@ -187,8 +190,11 @@ class BeardTemplateRendererSpec extends FunSpec with Matchers {
           renderer.render(template, renderResult, context(true))
           renderResult.result.toString should be("""
                                                    |<ul>
+                                                   |
```

Comment:
I don't think that this is the way the template should be rendered. The version without the new lines look more natural.